src/app/executions/tasks-runtime/tasks-runtime.component.ts

```diff
@@ -92,6 +105,18 @@ export class TasksRuntimeComponent implements OnInit {
                 preBarWidth = 99;
             }
             task.preBarWidth = preBarWidth;
+
+            const waitingBeforeBarWidth = (task.started_at_relative / workflowDurationSec) * 100;
+            task.waitingBeforeBarWidth = waitingBeforeBarWidth || 0;
```

Comment:
There is no benefit in doing `expr || 0` because 0 evaluates to `false`. So whatever the value of `waitingBeforeBarWidth` is, it is the one that will be taken.
So these 2 lines could be minified to
```js
task.waitingBeforeBarWidth = (task.started_at_relative / workflowDurationSec) * 100;
```

(the reason for the `runningBarWidth || 1` on line 114 is to have a minimum bar width of 1 when the running time is 0, for visual purposes)

Reply:
TY, now I understand